libs/EXTERNAL/libinterchange/interchange/DeviceResources.capnp

```diff
@@ -59,9 +59,8 @@ annotation wireRef(*) :WireRef;
 using WireIdx = UInt32;
 
 struct WireTypeRef {
-    type  @0 :Ref.ReferenceType = parent;
+    type  @0 :Ref.ReferenceType = rootValue;
```

Comment:
This file should be updated by following this instructions https://docs.verilogtorouting.org/en/latest/README.developers/#updating-an-existing-subtree

Reply:
Resolved in next push